### ISSUE TITLE
NES boot rom name

### ISSUE THREAD

**Issue by djvj1** [uberyoji/mister-boot-roms#2]
On my Mister NES needs to be named boot.rom to load. Might have to do with FDS boot roms in the folder as well.

**Comment by uberyoji** [comment]
Are you using the official nes core?

**Comment by djvj1** [comment]
Yes the one from the mister Downloader.

**Comment by uberyoji** [comment]
Can I have a snapshot of of the files on your sd in games/NES?
The proper naming should be this:
boot0.rom = FDS BIOS file. Will be used for any FDS images loaded
boot1.rom = NES Cart file. Can be used with boot0.rom (BIOS) in place
boot2.rom = FDS image file. Requires boot0.rom (BIOS). Use a blank FDS (header only) to boot the FDS BIOS without a disk image.
boot3.rom = PAL file. It can be used to set your default custom palette. Save the menu option on "Custom" to apply immediately.

**Comment by djvj1** [comment]
Yes that's how I have my files. If I rename boot2, the custom bios works.
if I rename boot0, I get a grey screen. If I rename boot1 to boot, it also
lets the custom bios work.

Attached the files

**Comment by uberyoji** [comment]
I dropped this path because Sorgelig mentionned many times it would no longer be supported in the future.
